init/runShadowsocks.js

```diff
@@ -18,16 +18,21 @@ const run = async () => {
   if(runParams.includes(':')) {
     method = runParams.split(':')[1];
   }
+  const pluginOptions = [
+    ...(config.shadowsocks['plugin'] ? ['--plugin', config.shadowsocks['plugin']] : []),
+    ...(config.shadowsocks['plugin-opts'] ? ['--plugin-opts', config.shadowsocks['plugin-opts']] : []),
+  ];
   let shadowsocks;
   if(runParams.includes('python')) {
     type = 'python';
     const tempPassword = 'qwerASDF' + Math.random().toString().substr(2, 8);
+    // there is no SIP003 support in the python port, so just ignore the pluginOptions
     shadowsocks = spawn('ssserver', ['-m', method, '-p', '65535', '-k', tempPassword, '--manager-address', config.shadowsocks.address]);
   } else if(runParams.includes('rust')) {
     type = 'rust';
-    shadowsocks = spawn('ssmanager', [ '-m', method, '-U', '--manager-address', config.shadowsocks.address]);
+    shadowsocks = spawn('ssmanager', [ '-m', method, '-U', '--manager-address', config.shadowsocks.address, ...pluginOptions]);
```

Comment:
I don't think `ssmanager` supports `--plugin` and `--plugin-opts` from command line options.

Reply:
Yes, you are right. I confused 'ssserver' and 'ssmanager' in shadowsocks-rust. 'ssserver' does accept plugin and plugin-opts parameters while 'ssmanager' not after my second check of the source code.

BTW, I reviewed the souce code of shadowsocks-libev. It does handles plugin and plugin-opts parameters in 'manager.c'.

So this line here is not effective. Only libev's manager has supported to handle plugin and plugin-opts.